3_visualize/src/report-do-inventory.Rmd

```diff
@@ -17,8 +17,7 @@ options(scipen=999)
 DO_wqp_data <- tar_read(p2_filtered_wqp_data_subset)
 
 # Load full site list:
-site_list_path <- tar_read(p2_site_list_csv)
```

Comment:
The leaflet map further down in this document calls `site_list_path` so in order for the pipeline to build, we either need to 

- adjust the `map_sites` function to take `site_list` instead of `site_list_csv` (maybe this is the better longer term solution; this function is in `3_visualize/map_sites.R), or
- revert lines 20-21 here (for site_list_path, could just replace with `p2_sites_w_segs_csv`, since that is the same site list but with the new segid variable)